.github/workflows/main.yaml

```diff
@@ -170,13 +165,6 @@ jobs:
           tags: sceptreorg/sceptre:${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
```

Comment:
There's a lot going on in this block of code. I think I would add a blank line before each of these keys in the jobs block

```
% yq '.jobs | keys' .github/workflows/main.yaml  
[
  "docker-build",
  "docker-build-push",
  "docker-build-release",
  "documentation",
  "linting",
  "packaging",
  "pypi-release",
  "unit-tests"
]
```

Reply:
done